openforcefield/topology/topology.py

```diff
@@ -1798,7 +1806,8 @@ def IsAmideBond(oe_bond):
 
         return topology, positions
 
-    def to_openeye(self,
+    # TODO: Jeff prepended an underscore on this before 0.2.0 release to remove it from the API. Fix?
```

Comment:
Why?

Reply:
Added 
```
    # TODO: Jeff prepended an underscore on this before 0.2.0 release to remove it from the API.
    #       This function is deprecated and expects the OpenEye toolkit. We need to discuss what
    #       to do with this functionality in light of our move to the ToolkitWrapper architecture.
    #       It also expects Topology to be organized by chain, which is not currently the case.
    #       Bringing this function back would require non-trivial engineering and testing, and we
    #       would want to discuss what "guarantee" of correctness it could offer.
```